reference/JsSetHostPromiseRejectionTracker.md

```diff
@@ -0,0 +1,22 @@
+Sets a callback function that will be called to notify of unhandled Promise Rejections.
```

Comment:
Can also be used to notify handled rejection. 

You can just cut & paste the first para of remarks here instead.